variables.tf

```diff
@@ -26,6 +26,22 @@ variable "policy" {
   description = "A bucket policy in JSON format."
   type        = string
   default     = null
+
+  # Example:
+  # policy + [
+  #
+  #     {
+  #       "Sid": "IPAllow",
+  #       "Effect": "Deny",
+  #       "Principal": "*",
+  #       "Action": "s3:*",
+  #       "Resource": "arn:aws:s3:::examplebucket/*",
+  #       "Condition": {
+  #          "NotIpAddress": {"aws:SourceIp": "54.240.143.0/24"}
+  #
+  #       }
+  #     }
```

Comment:
does not work like this.... needs a data source or string..

Reply:
dismissed